app/ssttpcalculator/CalculatorService.scala

```diff
@@ -417,9 +418,9 @@ object CalculatorService {
       if (initialPayment.equals(BigDecimal(0))) {
         val daysBetweenStartAndPaymentDates = DAYS.between(startDate, defaultPaymentDate)
 
-        if (daysBetweenStartAndPaymentDates < oneWeek && DAYS.between(startDate, defaultPaymentDatePlusOneMonth) < oneWeek)
+        if (daysBetweenStartAndPaymentDates < tenDays && DAYS.between(startDate, defaultPaymentDatePlusOneMonth) < oneWeek)
```

Comment:
Another reference to `oneWeek` here if adding a calendar month

Reply:
Yep, that was a bug. Thanks, now fixed